src/parser/csql_grammar.y

```diff
@@ -1652,6 +1652,8 @@ stmt_list
 			  }
 
 		DBG_PRINT}}
+        | ';'
```

Comment:
Is it for a quick fix?

Is it difficult to change the following?
It seems that the following are not simple changes.
(Errors related to CUBRID_STMT_TYPE, etc.)
```
stmt_list	
        : stmt_list stmt
        | stmt
        ;

stmt
        : stmt_
        | ';'
```

Reply:
The example provided is the format when stmts do not need to be separated by semicolons.

I didn't quite understand what you said about CUBRID_STMT_TYPE.